### PR TITLE
single draft or something

### PR DESCRIPTION
**Pull request by chrisinajar**

### CODE REVIEW

File: content/panorama/styles/custom_game/hero_selection.css [hunk 1]
```diff
@@ -105,13 +105,23 @@ Label {
   height: 100%;
 }
 
-#TeamRadiant {
-  flow-children: right;
+#TeamRadiant, #TeamDire {
```
Comment (DarkoniusXNG) [inline content/panorama/styles/custom_game/hero_selection.css:108]: In some patch, Valve broke the usage of commas and sometimes they don't work well with their resource compiler, so if you ever get some weird unexplainable error related to hero_selection.css, be sure it's this ...

File: game/scripts/vscripts/components/heroselection/heroselection.lua [hunk 2]
```diff
@@ -935,12 +974,34 @@ function HeroSelection:GetPreviewHero (playerId)
...
+function HeroSelection:RandomHeroByAbility (ability, heroExclusions)
+  local attempts = 0
+  while true do
+    attempts = attempts + 1
+    local choice = HeroSelection:RandomHero()
+    if not heroExclusions[choice] and herolist[choice] == ability then
+      return choice
+    end
+
+    if attempts > 1000 then
```
Comment (DarkoniusXNG) [inline game/scripts/vscripts/components/heroselection/heroselection.lua:986]: 1000 attempts? Lua lags even after 10 attempts and you have 1000 lmao. Maybe put max amount of heroes (124+) or number of available non-banned heroes.

Comment (chrisinajar) [reply game/scripts/vscripts/components/heroselection/heroselection.lua:986]: this is a really right loop, it was infinite before to be fair haha

File: content/panorama/styles/custom_game/hero_selection.css (hunk 1 above)
Comment (chrisinajar) [reply content/panorama/styles/custom_game/hero_selection.css:108]: lol dem fuks

**Pull request merged**